### PR TITLE
Damage updated to 6.23

### PR DESCRIPTION
**Pull request by myo**
compiles and works fine
why didn't admins update libs up until now Kappa

### CODE REVIEW

**Comment by YeezyYeezyWhatsGood** [comment]
L33T

File: Damage.cs [hunk 1]
```diff
@@ -2038,6 +2053,48 @@ static Damage()
                     });
 
             Spells.Add(
+                "AurelionSol",
```
Comment (YeezyYeezyWhatsGood) [inline Damage.cs]: XDDD

Comment (myo) [inline Damage.cs]: ye there was no aoshin :feelsbadman:

**Comment by YeezyYeezyWhatsGood** [comment]
@myo Rengar empowered missing!

**Comment by myo** [comment]
hoped you wouldn't notice :kappa: will add it with ivern brush buff thingy when i get home

**Comment by YeezyYeezyWhatsGood** [comment]
Downvoted

**Pull request merged**